src/js/components/List/List.js

```diff
@@ -245,7 +247,19 @@ const List = React.forwardRef(
             as={as || 'ul'}
             itemFocus={itemFocus}
             tabIndex={onClickItem || onOrder ? 0 : undefined}
-            onBlur={onOrder ? () => setActive(undefined) : undefined}
+            onBlur={
+              onOrder
+                ? () => {
+                    setActive(undefined);
+                  }
+                : () => {
+                    if (active) setLastActive(active);
```

Comment:
If `active` is 0, this won't be true. I think it's fine to always `setLastActive(active)` irrespective of what `active` is.

Reply:
After replacing, works as intended with the new updates to its own `onFocus`